test/spec/modules/openxBidAdapter_spec.js

```diff
@@ -476,11 +476,6 @@ describe('OpenxAdapter', () => {
       expect(result.length).to.equal(0);
     });
 
-    it('should register a user sync', () => {
-      spec.interpretResponse({body: bidResponse}, bidRequest);
-      sinon.assert.calledWith(userSync.registerSync, 'iframe', 'openx', 'http://testpixels.net');
-    });
-
     it('should register a beacon', () => {
```

Comment:
@haohany Please add unit test for getUserSync

Reply:
Updated with tests. Thanks for the review.